Makefile

```diff
@@ -54,6 +61,34 @@ build-image-ocp-example: build-base-image ## Build a rag-content container image
 build-base-image: ## Build base container image
 	podman build -t $(TORCH_GROUP)-road-core-base -f Containerfile.base --build-arg FLAVOR=$(TORCH_GROUP)
 
+start-postgres: ## Start postgresql from the pgvector container image
+	podman run -d --name pgvector --rm -e POSTGRES_PASSWORD=$(POSTGRES_PASSWORD) \
+	 -p $(POSTGRES_PORT):5432 \
+	 -v $(PWD)/postgresql/data:/var/lib/postgresql/data:Z pgvector/pgvector:pg16
+
+start-postgres-debug: ## Start postgresql from the pgvector container image with debugging enabled
+	mkdir -pv ./postgresql/data ./output
+	podman run --name pgvector --rm -e POSTGRES_PASSWORD=$(POSTGRES_PASSWORD) \
+	 -p $(POSTGRES_PORT):5432 \
+	 -v ./postgresql/data:/var/lib/postgresql/data:Z pgvector/pgvector:pg16 \
+	 postgres -c log_statement=all -c log_destination=stderr
+
+generate-embeddings-postgres: ## Generate embeddings for postgres vector store
```

Comment:
I've tried running the target as described in the README, and the process seems to hang, with minimal CPU usage. I'm not exactly sure if the problem isn't in my environment though.